community/examples/hpc-slurm6.yaml

```diff
@@ -37,51 +37,45 @@ deployment_groups:
     settings:
       local_mount: /home
 
-  - id: debug_nodeset
+  - id: debugnodeset
     source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
     use: [network]
     settings:
-      name: ns1
       node_count_dynamic_max: 4
       machine_type: n2-standard-2
       enable_placement: false  # the default is: true
 
-  - id: debug_partition
+  - id: debug
     source: community/modules/compute/schedmd-slurm-gcp-v6-partition
-    use: [debug_nodeset, homefs]
+    use: [debugnodeset, homefs]
     settings:
-      partition_name: debug
       exclusive: false  # allows nodes to stay up after jobs are done
       is_default: true
 
-  - id: compute_nodeset
+  - id: computenodeset
```

Comment:
this will be the name of the nodes, right? it would almost be nicer to have a shorter name.